sbpy/spectroscopy/tests/data/HCN.csv

```diff
@@ -0,0 +1 @@
+UT_Date,Time,T_B,log(Q),error_top,error_bottom,Q,Q_top,Q_bottom
2.0558,2010-11-02 01:20:21,0.672,24.813,0.081,0.013,6.5013E+24,7.8343E+24,6.30957E+24
2.1075,2010-11-02 02:34:48,1.06,25.011,0.078,0.007,1.02565E+25,1.22744E+25,1.00925E+25
2.1672,2010-11-02 04:00:46,1.234,25.076,0.078,0.004,1.19124E+25,1.42561E+25,1.18032E+25
2.2289,2010-11-02 05:29:37,1.357,25.118,0.082,0.004,1.3122E+25,1.58489E+25,1.30017E+25
2.2817,2010-11-02 06:45:39,1.89,25.262,0.077,0.004,1.8281E+25,2.18273E+25,1.81134E+25
2.3359,2010-11-02 08:03:42,1.786,25.237,0.081,0.007,1.72584E+25,2.0797E+25,1.69824E+25
2.3723,2010-11-02 08:56:07,1.509,25.164,0.087,0.012,1.45881E+25,1.78238E+25,1.41906E+25
3.0339,2010-11-03 00:48:49,1.222,25.072,0.077,0.008,1.18032E+25,1.40929E+25,1.15878E+25
3.0868,2010-11-03 02:05:00,1.503,25.162,0.079,0.005,1.45211E+25,1.74181E+25,1.43549E+25
3.1495,2010-11-03 03:35:17,1.916,25.268,0.078,0.004,1.85353E+25,2.2182E+25,1.83654E+25
3.2062,2010-11-03 04:56:56,1.72,25.221,0.084,0.004,1.66341E+25,2.01837E+25,1.64816E+25
3.2593,2010-11-03 06:13:24,2.154,25.319,0.083,0.004,2.08449E+25,2.52348E+25,2.06538E+25
3.302,2010-11-03 07:14:53,1.869,25.257,0.08,0.005,1.80717E+25,2.1727E+25,1.78649E+25
3.3527,2010-11-03 08:27:53,1.115,25.032,0.082,0.005,1.07647E+25,1.30017E+25,1.06414E+25
4.0268,2010-11-04 00:38:36,1.595,25.188,0.077,0.007,1.5417E+25,1.84077E+25,1.51705E+25
4.0676,2010-11-04 01:37:21,1.775,25.234,0.077,0.005,1.71396E+25,2.04644E+25,1.69434E+25
4.1222,2010-11-04 02:55:58,1.577,25.183,0.081,0.005,1.52405E+25,1.83654E+25,1.50661E+25
4.1852,2010-11-04 04:26:41,1.186,25.059,0.083,0.005,1.14551E+25,1.38676E+25,1.1324E+25
4.2492,2010-11-04 05:58:51,0.889,24.934,0.085,0.01,8.59014E+24,1.04472E+25,8.3946E+24
4.2904,2010-11-04 06:58:11,0.978,24.976,0.085,0.009,9.46237E+24,1.1508E+25,9.2683E+24
4.3399,2010-11-04 08:09:27,0.753,24.862,0.087,0.011,7.2778E+24,8.89201E+24,7.09578E+24
5.0163,2010-11-05 00:23:28,0.795,24.886,0.088,0.013,7.6913E+24,9.4189E+24,7.46449E+24
5.0673,2010-11-05 01:36:55,0.762,24.867,0.078,0.011,7.36207E+24,8.81049E+24,7.17794E+24
5.1246,2010-11-05 02:59:25,0.67,24.811,0.076,0.009,6.47143E+24,7.70903E+24,6.3387E+24
5.1873,2010-11-05 04:29:43,0.521,24.702,0.082,0.013,5.03501E+24,6.08135E+24,4.88652E+24
5.2426,2010-11-05 05:49:21,0.578,24.747,0.081,0.015,5.5847E+24,6.72977E+24,5.39511E+24
5.2932,2010-11-05 07:02:12,1.009,24.989,0.084,0.009,9.7499E+24,1.18304E+25,9.54993E+24
5.3477,2010-11-05 08:20:41,1.216,25.07,0.083,0.015,1.1749E+25,1.42233E+25,1.13501E+25
```

Comment:
This file uses a ^M return character, can you replace it by `\n` and amend the commit ?

Reply:
How was the observing time parsed from the table in the paper? If this can be done using astropy maybe useful to have as part of the test function.

Reply:
Will do. The observing time I did the conversion directly in excel. I can check if astropy.time supports the time format used in the paper.